packages/desktop/src/renderer/app/components/environment-logs/environment-logs.component.html

```diff
@@ -328,6 +328,9 @@
                   <div class="environment-logs-content-item">
                     <strong>Status:</strong> {{ selectedLog.response?.status }}
                   </div>
+                  <div class="environment-logs-content-item">
+                    <strong>StatusMessage:</strong> {{ selectedLog.response?.statusMessage }}
```

Comment:
Maybe title could be more human readable:
```suggestion
                    <strong>Status message:</strong> {{ selectedLog.response?.statusMessage }}
```